travelplan/src/SearchPage/Constants.js

```diff
@@ -0,0 +1,2 @@
+export const SIGN_IN = "Sign in";
```

Comment:
Change the naming of the file into the start of lower case like the following: `constants.js`